aws/lambdas/readEventPage/index.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
```

Comment:
It is a bit misleading because I was trying to find the page that tells us the most about the event and it is easy to parse the information from. Since the matches page has the name of the league and the year or the event I am using that, but the main purpose of this function is to get the event information. It is only an additional plus that it has the matches as well. We could rename it to readEventDetails, how about that?